manifest.php

```diff
@@ -24,11 +24,11 @@
     'label' => 'extension-tao-outcomerds',
     'description' => 'extension that allows a storage in relational database',
     'license' => 'GPL-2.0',
-    'version' => '4.3.0',
+    'version' => '4.3.1',
```

Comment:
Same here, if u bump major version of dependency, please update manifest version

Reply:
Done